resources/testdriver.js

```diff
@@ -1078,6 +1126,19 @@
          */
         in_automation: false,
 
+        bidi: {
+            log: {
+                entry_added: {
+                    subscribe: function () {
```

Comment:
See previous comment about where the `async` goes. Adding in the `params` argument so this is easy to copy-paste. Also, for some reason, the style is different in `test_driver_internal`. All in all:

```suggestion
                    async subscribe(params) {
```